.github/workflows/ci.yml

```diff
@@ -62,6 +55,7 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v2
       with:
+        driver: docker
```

Comment:
This lets us cache build artifacts on local Docker daemon instead of more complex buildkit caching.